README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
Consider maintaining a consistent style for the README. If these lines are meant for testing purposes, it's recommended to remove them from the main README and use a separate test file or process instead.